tests/ensemble_evaluator/test_prefect_ensemble.py

```diff
@@ -23,20 +23,55 @@
 
 
 def parse_config(path):
-    conf_path = os.path.abspath(path)
+    conf_path = Path(path).resolve()
     with open(conf_path, "r") as f:
         return yaml.safe_load(f)
 
 
 @pytest.mark.timeout(60)
 def test_run_prefect_ensemble(tmpdir, unused_tcp_port):
+    with tmp(Path(SOURCE_DIR) / "test-data/local/prefect_test_case", False):
```

Comment:
Is not this something one could do with the tmpdir argument instead? :thinking:

Reply:
Yes, that is a bit weird, looks like the `tmpdir` fixture from `pytest` and our own `tmp` context manager are being mixed, which is probably a bad idea. I would say that in new code we strive to use the `tmpdir` fixture. I will look into it.